pages/samvera/developer_resources/collections/hyrax_2.1/participants.md

```diff
@@ -0,0 +1,203 @@
+---
+title: "FAQ - Collection Sharing Participants"
+keywords: Participants, Collection, FAQ, Sharing
+categories: How to Do All the Things
+permalink: collection_extensions_faq_participants.html
+folder: samvera/how-to/collection_extensions/hyrax_2.1/faq_participants.md
+sidebar: home_sidebar
+tags: [development_resources]
+a-z: ['FAQ - Collection Sharing Participants', 'Collection Extensions - Sharing Participants', 'Sharing Participants for Collections', 'Participants for Collections']
+version:
+  label: 'Hyrax v2.1.0'
+  branch:
+    label: 'collections-sprint'
+    link: 'https://github.com/samvera/hyrax/tree/collections-sprint'
+---
+
+## Understanding Collection Type Participants
+
+NOTE: Only admins can create, edit, and delete collection types.  Participants set for a collection type effect how users can interact with collections of this type.
+
+### How to set participants for a collection type?
+
+* Dashboard -> Settings -> Collection Types
+* Click Edit beside the collection type you want to update
+* Select Participants tab
+* Search for user/group
+* Set access level to Manager or Creator
+* Click Add button
+
+### Manager
+
+**Description:** 
+
+Managers for a collection type can edit collections other users created, including adding to and removing works from a collection, modifying collection metadata, and deleting collections, only for collections of the type they manage.
+
+**Default:**  Repository Administrators
+
+**Implementation:**
+
+* When a manager is added, an entry is added to `collection_type_participants` database table granting the user/group :manage access.
+* When a manager is removed, the entry in `collection_type_participants` database table for this manager is removed.
+* When the create new collection process is initiated, the user is allowed to create collections of types for which they have manage access.
+* When a collection is created of a type, each collection type manager is made a manager of the collection which grants them `edit_access` to the new collection regardless of which user creates the collection.  This `edit_access` grant is what grants managers the abilities to edit the collection, collection items, and metadata.
+  
+NOTE: Access to a collection is granted when the collection is created.  Changes to managers after a collection is created do not change the access grants of the collection.
+
+Special Note on Admin Sets:
+* Admin Set collection type is pre-defined and has default managers assigned.
+* **Default:**  Administrators
+* A site can assign managers to the Admin Set collection type.  This grants them :manage access to admin sets created after they became a manager.  At the collection type level, assigning managers has the same effect as for other collection types.
+
+
+### Creator
+
+**Description:** 
+
+Creators for a collection type can create collections of this type.  They are given manage acceess only to collections they create.
```

Comment:
typo on "acceess"

Reply:
The file with this typo moved to collection-sharing.md.  The typo was fixed there.